public/style/404.css

```diff
@@ -0,0 +1,48 @@
+html, body {
+  height: 100%;
+  overflow: hidden;
+  background-color: #343a40;
+}
+
+.error-page {
+  display: flex;
+  width: 80%;
+  height: 95%;
+  margin: auto;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  vertical-align: center;
+  border: 3px solid white;
+  margin-top: 10px;
+}
+
+.error-page h1 {
+  font-size: 20vh;
+  font-weight: bold;
+  margin: -6vh 0 0;
+  color:white;
```

Comment:
Please add space after ':'. And you can use #fff instead of 'white' like in line 4